xml/ha_storage_protection.xml

```diff
@@ -698,6 +698,63 @@ Timeout (msgwait)  : 10
    </procedure>
   </sect1>
 
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting Up Diskless SBD</title>
+   <para>SBD can be operated in a diskless mode.</para>
+    <important>
+     <title>Number of Cluster Nodes</title>
+       <para>
+         Do <emphasis>not</emphasis> use diskless SBD as fencing mechanism
+         for two-node clusters. Use it only in clusters with three or more
+         nodes. SBD in diskless mode cannot handle split brain
+         scenarios for two-node clusters.
+      </para>
+   </important>
+   <para>
+    When setting up SBD, you need to take several timeout values into account.
+    For details, see <xref linkend="sec.ha.storage.protect.watchdog.timings"/>.
+   </para>
+
+   <procedure xml:id="pro.ha.storage.protect.confdiskless">
+    <title>Configuring Diskless SBD</title>
+    <step>
+     <remark>toms 2018-04-25: Move the para and screen into section about
+     1-3 devices too?</remark>
```

Comment:
I'd prefer separate sections for "Normal SBD" and "Diskless SBD" to prevent any confusions :)

Reply:
@gao-yan : It actually is in a separate section, which starts above this procedure.
However, what we did not get right is that diskless SBD needs no fencing resource. Thus I'm currently adjusting  the structure  in a new branch (new PR is coming soon). We also will attach a DRAFT PDF version for you to review.